media/css/firefox/enterprise.scss

```diff
@@ -5,6 +5,7 @@
 @import '../pebbles/includes/lib';
 @import '../hubs/sections';
 @import '../hubs/masthead';
+@import '../hubs/nav-download-button-remove';
```

Comment:
Note for reviewer: I decided to hide the download button in the nav for this page, since it does not result in an enterprise edition download.

Reply:
💡